### DISCUSSION TITLE
2.6.1 is available to download

### DISCUSSION THREAD

**Discussion by mekya** [ant-media/Ant-Media-Server#5187] [Announcements]
Hey Guys,

2.6.1 is available to download on [Github Releases](https://github.com/ant-media/Ant-Media-Server/releases/tag/ams-v2.6.1) for Community Edition and antmedia.io for Enterprise Edition. 

Blog post mentioning highlights in 2.6.0 and 2.6.1 will be available on [antmedia.io](https://antmedia.io)

Changelog  
<h3>Ant-Media-Server</h3>
<li>
    <a href='https://github.com/ant-media/Ant-Media-Server/pull/5179'>5179</a>
    Remove redundant log - <a href=''></a>
</li>
<li>
    <a href='https://github.com/ant-media/Ant-Media-Server/pull/5155'>5155</a>
    Stream high availability - <a href='https://github.com/ant-media/Ant-Media-Server/issues/4730'>https://github.com/ant-media/Ant-Media-Server/issues/4730</a>
</li>
<li>
    <a href='https://github.com/ant-media/Ant-Media-Server/pull/5145'>5145</a>
    Community hls playback - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5128'>https://github.com/ant-media/Ant-Media-Server/issues/5128</a>
</li>
<li>
    <a href='https://github.com/ant-media/Ant-Media-Server/pull/5140'>5140</a>
    Add relative file path to the imported vods - <a href='https://github.com/ant-media/Ant-Media-Server/issues/4866'>https://github.com/ant-media/Ant-Media-Server/issues/4866</a>
</li>
<li>
    <a href='https://github.com/ant-media/Ant-Media-Server/pull/5132'>5132</a>
    Support ticket socket timeouts increased for large log file transfers - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5088'>https://github.com/ant-media/Ant-Media-Server/issues/5088</a>
</li>
<li>
    <a href='https://github.com/ant-media/Ant-Media-Server/pull/5131'>5131</a>
    Fix typo in link for docker in Readme  - <a href=''></a>
</li>
<li>
    <a href='https://github.com/ant-media/Ant-Media-Server/pull/5129'>5129</a>
    Release/2.6.0 - <a href=''></a>
</li>
<li>
    <a href='https://github.com/ant-media/Ant-Media-Server/pull/5127'>5127</a>
    Update README - <a href=''></a>
</li>
        <h3>Ant-Media-Enterprise</h3>
        <li>
            Use nginx in cluster test &Fix app distribution- <a href=https://github.com/ant-media/Ant-Media-Server/issues/5167>https://github.com/ant-media/Ant-Media-Server/issues/5167</a>
        </li>
        <li>
            Add test code for mobile playback- <a href=></a>
        </li>
        <li>
            Add test code for vod play and one time token- <a href=></a>
        </li>
        <li>
            Release/2.6.0- <a href=></a>
        </li>
        <li>
            Update selenium-java and webdrivermanager version- <a href=></a>
        </li>
        <h3>StreamApp</h3>
        <li>
            <a href='https://github.com/ant-media/StreamApp/pull/370'>370</a>
            Fix soundmeter and Stream high availability - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5165'>https://github.com/ant-media/Ant-Media-Server/issues/5165</a>
        </li>
        <li>
            <a href='https://github.com/ant-media/StreamApp/pull/369'>369</a>
            Fix vod playback and mobile playback - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5144'>https://github.com/ant-media/Ant-Media-Server/issues/5144</a>
        </li>
        <li>
            <a href='https://github.com/ant-media/StreamApp/pull/367'>367</a>
            Fix reconnection issues - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5141'>https://github.com/ant-media/Ant-Media-Server/issues/5141</a>
        </li>
        <li>
            <a href='https://github.com/ant-media/StreamApp/pull/366'>366</a>
            Support old vod playback structure for compatibility - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5137'>https://github.com/ant-media/Ant-Media-Server/issues/5137</a>
        </li>
        <li>
            <a href='https://github.com/ant-media/StreamApp/pull/365'>365</a>
            Add logo option as a comment - <a href='https://github.com/ant-media/Ant-Media-Server/issues/3651'>https://github.com/ant-media/Ant-Media-Server/issues/3651</a>
        </li>
        <li>
            <a href='https://github.com/ant-media/StreamApp/pull/364'>364</a>
            Release/2.6.0 - <a href=''></a>
        </li>
        <li>
            <a href='https://github.com/ant-media/StreamApp/pull/363'>363</a>
            update adapter-latest.js to latest version issue#5050 - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5050'>https://github.com/ant-media/Ant-Media-Server/issues/5050</a>
        </li>
        <h3>Ant-Media-Server-Parent</h3>
        <li>
            <a href='https://github.com/ant-media/ant-media-server-parent/pull/100'>100</a>
            Add selenium, webdriver and netty version - <a href=''></a>
        </li>
        <li>
            <a href='https://github.com/ant-media/ant-media-server-parent/pull/99'>99</a>
            Release/2.6.0 - <a href=''></a>
        </li>
        <h3>Ant-Media-Management-Console</h3>
        <li>
            <a href='https://github.com/ant-media/Ant-Media-Management-Console/pull/222'>222</a>
            update docs links and add few now links to docs - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5152'>https://github.com/ant-media/Ant-Media-Server/issues/5152</a>
        </li>
        <li>
            <a href='https://github.com/ant-media/Ant-Media-Management-Console/pull/220'>220</a>
            Fix vod play with different scenarios - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5137'>https://github.com/ant-media/Ant-Media-Server/issues/5137</a>
        </li>
        <li>
            <a href='https://github.com/ant-media/Ant-Media-Management-Console/pull/211'>211</a>
            Bump jszip from 3.7.1 to 3.10.1 - <a href=''></a>
        </li>
        <li>
            <a href='https://github.com/ant-media/Ant-Media-Management-Console/pull/209'>209</a>
            Bump ua-parser-js from 0.7.31 to 0.7.33 - <a href=''></a>
        </li>
        <li>
            <a href='https://github.com/ant-media/Ant-Media-Management-Console/pull/202'>202</a>
            Bump json5 from 1.0.1 to 1.0.2 - <a href=''></a>
        </li>
        <li>
            <a href='https://github.com/ant-media/Ant-Media-Management-Console/pull/199'>199</a>
            Bump jsonwebtoken from 8.5.1 to 9.0.0 - <a href=''></a>
        </li>

**Comment by fabiox23** [comment]
Hi @mekya, that's great but I cannot find a download link for version 2.6.1 of the Enterprise Edition on my antmedia.io account. I only have 2.5.x. Can somebody help please? Many thanks.

**Reply by yashtandon113** [reply]
Hi @fabiox23 
Can you please share your email Id that is registered on antmedia.io?

**Reply by mekya** [reply]
We're getting this problem from time to time. Let's have a permanent resolution to not encounter this again.  
Could you please create an issue?

**Comment by fabiox23** [comment]
Hi @yashtandon113 , how can I send you a private message with my email id?

**Reply by mekya** [reply]
Hi @fabiox23, Just send an email to support@antmedia.io and the team will handle that.
